README.md

```diff
@@ -34,7 +34,7 @@ For an example receiver dApp, please refer to the [ExampleReceiverDapp](./src/ex
 
 4. You can verify the senderLine and senderDapp addresses when reciving the message.
 
-    For an example, please check [Verify example](https://github.com/darwinia-network/darwinia-msgport/blob/main/contracts/examples/ExampleReceiverDapp.sol#L20)
+    For an example, please check [Verify example](./src/examples/ExampleReceiverDapp.sol#L36)
```

Comment:
Suggest removing `L36`, the content of this file maybe changes in the future, jump to the file level is enough to me.